### ISSUE TITLE
Embedded OpenType (.eot) font files in archives not unpacking correctly

### ISSUE THREAD

**Issue by swinslow** [fossology/fossology#915]
Issue:
Embedded OpenType (.eot) font files included in FOSSology 3.1.0 uploads are apparently not being unpacked correctly. Trying to view an .eot file in the License Browser results in the viewer instead skipping back to view the first file in that scan. The .eot file cannot be viewed or have license results set.

Steps to reproduce:
1. In FOSSology 3.1.0, upload an archive that includes an .eot file. A sample archive is attached as [eot_example.tar.gz](https://github.com/fossology/fossology/files/1359619/eot_example.tar.gz).
2. View the corresponding scan in the License Browser.
3. Click on the sample .eot file "opensans-regular.eot". 

Expected result:
The contents of opensans-regular.eot should be viewable, and you should be able to set the corresponding license.
Alternatively, if .eot files are being treated as containers of sub-files (like PDFs or tar/zip files), then the sub-files within the .eot file should be viewable.

Actual result:
The contents of opensans-regular.eot cannot be viewed in the License Browser, either as a file or as a container. Instead, the License Browser will skip forward to the first regular unpacked file in the upload (in the attached example, this is file1.txt).

Additionally, the .eot file does not appear in an SPDX export of the scan results.

A screenshot of the FOSSology view is attached below:

<img width="1324" alt="fossology-view" src="https://user-images.githubusercontent.com/678569/31232108-d0d59c3c-a9b6-11e7-850b-113d0e4b3101.png">

**Comment by NicolasToussaint** [comment]
I have the same behaviour for EOT and OTF files.

For info, the ununpack agent yields the following logs:

```
Command 7z failed on: /srv/fossology/(...)fontawesome-webfont.eot
```

Full logs:
```
2017-12-29 15:00:31 ununpack [0] :: JOB[37].ununpack[6930.localhost]: "ERROR traverse.c.213: pfile 4187 Command 7z failed on: /srv/fossology/repository/debian-jessie-fossology/ununpack/165035l/5ae3c5086d4c9994979091060fbf2555bfbe9216.e13f147a5986ce8a9ba245c6f45a3e81.10240000.dir/docs/fonts/fontawesome-webfont.eot"
2017-12-29 15:00:31 ununpack [0] :: JOB[37].ununpack[6930.localhost]: "ERROR traverse.c.213: pfile 4187 Command 7z failed on: /srv/fossology/repository/debian-jessie-fossology/ununpack/165035l/5ae3c5086d4c9994979091060fbf2555bfbe9216.e13f147a5986ce8a9ba245c6f45a3e81.10240000.dir/docs/fonts/FontAwesome.otf"
```

**Comment by jeremiah** [comment]
It may be that skipping over files that cannot be ununpack'd is the way to go. It looks like .eot files are not able to be decompressed with FOSS tooling nor even are an open standard: https://en.wikipedia.org/wiki/Embedded_OpenType

**Comment by mcjaeger** [comment]
right, what we also do is to just remove these files before uploading.

**Comment by shaheemazmalmmd** [comment]
in fossology 4.5.0 we can add the extension of this (.eot) files in admin > Customize and select a option in upload page "Ignore scm files and files with specific mimetype to ignore them."
